docs/components/multiselect.md

```diff
@@ -57,6 +57,50 @@ An array of items containing a `label` and `value`. You may also specify:
 />
 ```
 
+### enableItemCreation
+
+`KMultiselect` offers the ability to add custom items to the list by typing the `label` you want to add and pressing `Enter` (which will automatically select it). Newly created items will have a randomly generated id for the `value` to ensure uniqueness. This action triggers an `added` event containing the newly added item data. Deselecting the item will completely remove it from the list. This action triggers a `removed` event containing the removed item's data.
+
+<ClientOnly>
+  <KLabel>Value:</KLabel> {{ mySelections }}
```

Comment:
suggestion: can you update all the instances in this docs file to display the `Value` object similarly to what we do in KTreeSelect and KDatetimePicker? I know it's a little out of scope, but it looks **so** nice ✨  

https://github.com/Kong/kongponents/blob/2e3642b9485191fb942ca46b7f2861a242b9126e/docs/components/tree-list.md#L25

You'll also need this style block:
```
pre.json {
  font-size: var(--type-sm);
  white-space: pre-wrap;
  padding: 16px;
  background-color: var(--grey-200);
  border-radius: 8px;
}
```

We use this style block in a couple components; you could add it to the `docs/.vitepress/theme/index.scss` file instead so it's reusable and then delete all the other occurrences

Reply:
I'm also guilty of duplicating this `pre` CSS style block. Three instances of it: two `pre.json` classes and one `pre.emitted-value`  (datetime-picker, file-upload, tree-list)

Reply:
I'll add it to the `index.scss` and clear up the other instances 👍🏻